content/en/blog/2024/collecting-file-based-java-logs-with-opentelemetry/index.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Collecting file-based Java logs with OpenTelemetry
+linkTitle: Collecting file-based Java logs with OpenTelemetry
```

Comment:
The `linkTitle` should be short and only be used if it is different from the `title`.

I also suggest that you re-think the title. Right now without knowing the content I would assume that this is just some docs on any kind of java logs with otel, but with context I understand that the crucial part is that there is a way to emit otel-compliant json logs and that can be read by the collector with ease

Reply:
Thanks @svrnm , @zeitlinger what do you think of "Collecting OpenTelemetry-compliant file-based Java logs"?

Reply:
If it doesn't change the meaning, another option that avoids the double hyphenated modifiers could be:
```suggestion
title: Collecting OpenTelemetry-compliant Java logs from files
```